bigbluebutton-html5/package-lock.json

```diff
@@ -401,6 +427,19 @@
         "concat-map": "0.0.1"
       }
     },
+<<<<<<< HEAD
+=======
```

Comment:
Committed unmerged file. Which is the correct way to resolve this? @BobakOftadeh

Reply:
Resolved in https://github.com/bigbluebutton/bigbluebutton/commit/cbb4bbc2fc0bec5bc4d6865ad9dcda4853e38bae